### ISSUE TITLE
Can`t open bCNC via .exe

### ISSUE THREAD

**Issue by Henrikastro** [vlachoudis/bCNC#1683]
So I was trying to use the makefile to compile the .exe of the bCNC and for some reason when I try to open it from there it gives me de error:
![Sem título](https://user-images.githubusercontent.com/37155325/155144926-d4099ca5-080f-444f-9d5f-b01a5dbac64c.png)
 and the Utils.py is definitly in the folder. I tried downgrading to python 2.7 but couldnt make it work. Also, if I try to open bCNC via cmd it works just fine.

**Comment by Nitsan-Suron** [comment]
Hey,
did you ever solved the problem?

**Comment by Henrikastro** [comment]
Not at all. Moved to a better GCode sender software called IoSender along with his GrblHAL
